### ISSUE TITLE
Wrong loss function

### ISSUE THREAD

**Issue by Mixnikon108** [esantamariavazquez/EEG-Inception#3]
The loss function in line 125  has been set a binary crossentropy, which is incorrect. According to the original paper, it should be categorical_crossentropy.
